jdi-light-mobile-tests/src/test/java/nativeapp_android/MobileFileManagerInit.java

```diff
@@ -0,0 +1,77 @@
+package nativeapp_android;
+
+import com.epam.jdi.light.driver.WebDriverFactory;
+import com.epam.jdi.light.mobile.elements.common.AppManager;
+import nativeapp.android.AndroidFileManager;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+import java.io.*;
+import java.nio.file.Files;
+
+import static com.epam.jdi.light.mobile.elements.init.PageFactory.initMobile;
+import static com.epam.jdi.light.settings.WebSettings.logger;
+import static nativeapp.android.AndroidFileManager.*;
+
+public class MobileFileManagerInit {
+
+    protected byte[] createdFileByte;
+    protected File createdFile;
+    protected File pulledFile;
+    public static final String FILE_NAME_TEST = "TEST.txt";
+    protected static final String PATH_TO_FILE_ANDROID = "/sdcard/download/" + FILE_NAME_TEST;
+    protected static final String PATH_TO_ANDROID_FOLDER = "/sdcard/download";
```

Comment:
Codacy found an issue: [Do not hardcode /sdcard.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6361799)